tests/integration/kubernetes/gha-run.sh

```diff
@@ -291,6 +291,33 @@ function deploy_nydus_snapshotter() {
 	echo "::endgroup::"
 }
 
+function cleanup_snapshotter() {
+	echo "::group::Cleanuping ${SNAPSHOTTER}"
+	case ${SNAPSHOTTER} in
+		nydus) cleanup_nydus_snapshotter ;;
+		*) >&2 echo "${SNAPSHOTTER} flavour is not supported"; exit 2 ;;
+	esac
+	echo "::endgroup::"
+}
+
+function cleanup_nydus_snapshotter() {
+	echo "cleanup_nydus_snapshotter"
+	local nydus_snapshotter_install_dir="/tmp/nydus-snapshotter"
+	if [ ! -d "${nydus_snapshotter_install_dir}" ]; then
+		>&2 echo "nydus snapshotter dir not found"
+		exit 1
+	fi
+	
+	pushd "$nydus_snapshotter_install_dir"
+	kubectl delete -f "nydus-snapshotter.yaml"
```

Comment:
Don't forget those yamls are in `misc/snapshotter/`

Reply:
Fixed.